pkg/motor/x/logger/utils.go

```diff
@@ -0,0 +1,35 @@
+package logger
+
+func conevertLogLevelCategory(level LogLevel) string {
+	switch level {
+	case LEVEL_INFO:
+		return "[INFO]"
+	case LEVEL_DEBUG:
+		return "[DEBUG]"
+	case LEVEL_WARN:
+		return "[WARN]"
+	case LEVEL_ERROR:
+		return "[ERROR]"
+	case LEVEL_FATAL:
+		return "[FATAL]"
+	}
+
+	return ""
+}
+
+func conevertLogLevel(level string) LogLevel {
```

Comment:
Where does the string version of these levels come from?

Reply:
it comes from motor as to allow for strings in the request for initalize